tests/integration/helpers/factories/db/invenio_records.py

```diff
@@ -81,25 +81,24 @@ def create_from_kwargs(cls, index=True, has_pid=True, **kwargs):
         instance.record_metadata = super(TestRecordMetadata, cls)\
                 .create_from_kwargs(updated_kwargs)
 
-        if index:
+        if index_name:
             instance.es_index_result = es.index(
-                index='records-hep',
-                doc_type='hep',
+                index=index_name,
+                doc_type=index_name.split('-')[-1],
                 body=instance.record_metadata.json,
                 params={}
             )
-            instance.es_refresh_result = es.indices.refresh('records-hep')
-
-        if has_pid:
-            instance.persistent_identifier = TestPersistentIdentifier\
-                    .create_from_kwargs(
-                        object_uuid=instance.record_metadata.id,
-                        pid_value=instance.record_metadata.json.get('control_number')
-                    ).persistent_identifier
+            instance.es_refresh_result = es.indices.refresh(index_name)
+
+        instance.persistent_identifier = TestPersistentIdentifier\
```

Comment:
heads-up: we will always create a ``pid``, do we really want this?

Reply:
We asked ourself the same question. I believe it makes sense to have a PIS every time a record is created. Also, this is cheap with a properly isolated app (PR on it's way)